### PR TITLE
No label_name and label_value in doc

### PR DESCRIPTION
**Pull request by mengqiy**

### CODE REVIEW

File: site/book/04-using-functions/02-imperative-function-execution.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ For example, the following will set the labels on all resources in wordpress pac
 the `Kptfile`:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources wordpress -- label_name=color label_value=orange
+$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources wordpress -- color=orange
```
Comment (mikebz) [inline site/book/04-using-functions/02-imperative-function-execution.md:139]: I would still keep two labels here, it's good to show that.  how about color=orange org=shipping

Comment (mengqiy) [reply site/book/04-using-functions/02-imperative-function-execution.md]: Done

**Review by mikebz — APPROVED** [review]
LGTM

File: site/book/04-using-functions/02-imperative-function-execution.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ For example, the following will set the labels on all resources in wordpress pac
 the `Kptfile`:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources wordpress -- label_name=color label_value=orange
+$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources wordpress -- color=orange fruit=apple
```
Comment (frankfarzan) [inline site/book/04-using-functions/02-imperative-function-execution.md:139]: let's choose more realistic lablels here:

`app: wordpress`
`env: prod`

**Comment by mengqiy** [comment]
PTAL

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**